views-fieldset/src/main/java/io/micronaut/views/fields/Form.java

```diff
@@ -25,8 +29,41 @@
  * @param action Form Action
  * @param method Form Method. For example `post`
  * @param fieldset Form fields
+ * @param enctype how the form-data should be encoded when submitting it to the server
  */
 @Experimental
 @Introspected
-public record Form(String action, String method, Fieldset fieldset) {
+public record Form(@NonNull String action,
+                   @Nullable @Pattern(regexp = "get|post") String method,
+                   @NonNull Fieldset fieldset,
+                   @Nullable @Pattern(regexp = "application/x-www-form-urlencoded|multipart/form-data|text/plain") String enctype) {
```

Comment:
Should we have a test to check this pattern is correct?

Reply:
I added a [validation tests]( https://github.com/micronaut-projects/micronaut-views/pull/676/commits/bc9d72b2222111e568a4e400fc9d6968eed73cc5). Don't hesitate to reject PRs and request changes such as this one.